extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -323,30 +340,48 @@ export class VirtualListCell<ItemInfo extends VirtualListCellInfo> extends RX.Co
 
         return (
             <RX.Animated.View
-                style={ [_styles.cellView, overflow, this._animatedStylePosition] }
+                style={ [_styles.cellView, overflow, this._animatedStylePosition, this._animatedStyleWidth] }
                 ref={ _virtualCellRef }
                 tabIndex={ this.props.tabIndex }
                 onLayout={ this.props.onLayout ? this._onLayout : undefined }
                 onFocus={ this.props.onCellFocus ? this._onFocus : undefined }
                 onBlur={ this.props.onCellFocus ? this._onBlur : undefined }
+                onPress={this.props.onItemSelected ? this._onPress : undefined}
+                onKeyPress={ this.props.onKeyPress || this.props.onItemSelected ? this._onKeyPress : undefined}
             >
                 <VirtualListCell.StaticRenderer
                     shouldUpdate={ this.props.shouldUpdate }
                     isFocused={ this.props.isFocused }
-                    style={ [overflow, this._animatedStyleWidth] }
+                    isSelected={ this.props.isSelected }
                     item={ this.props.item }
                     renderItem={ this.props.renderItem }
                 />
             </RX.Animated.View>
         );
     }
 
+    private _onKeyPress = (e: RX.Types.KeyboardEvent) => {
+        const isSelectItemKeyPress = e.keyCode === _keyCodeEnter || e.keyCode === _keyCodeSpace;
+        if (isSelectItemKeyPress && this.props.onItemSelected && this.props.item) {
+            this.props.onItemSelected(this.props.item);
+        }
```

Comment:
What if an item is not meant to be selectable, like the static text header for a group of items? Can the user simply not provide an onItemSelected callback in that case? Looks like that's the intent. We should make sure that's documented.

Reply:
I went back and forth on this one a little bit. It'll still call back onItemSelected & the listener can choose to ignore it.  I considered changing the isNavigable prop to be isSelectable to bring clickable & focusable elements inline, but I'm not entirely sure if that's desirable